src/braket/experimental/auxilary_functions/random_circuit/random_circuit.py

```diff
@@ -0,0 +1,54 @@
+import inspect
+import math
+import random
+from typing import List
+
+from braket.circuits import Circuit, Gate, Instruction
+from braket.circuits.gates import CNot, H, S, T
+
+
+def random_circuit(
+    num_qubits: int, num_gates: int, gate_set: List[Gate] = [CNot, S, T, H], seed=None
```

Comment:
nit - best practice is not to use mutable variables as default parameter values - because if you modify `gate_set` within the body of the function it will actually modify the default parameter for future calls to `random_circuit`.

Recommended way to do this is to give `gate_set` a default parameter value of `None`, and then in the body of the function have something like
```
if not gate_set:
    gate_set = [CNot, S, T, H]
```
or even more clever
```
gate_set = gate_set or [CNot, S, T, H]
```

Reply:
fixed, use first option for clarity